stdlib/public/stubs/Stubs.cpp

```diff
@@ -44,6 +46,9 @@
 #if defined(__CYGWIN__) || defined(__HAIKU__)
 #include <sstream>
 #endif
+
+#if SWIFT_STDLIB_HAS_LOCALE
+#include <clocale>
 #if defined(__OpenBSD__) || defined(__ANDROID__) || defined(__linux__) || defined(__wasi__) || defined(_WIN32)
```

Comment:
I don't understand the point of this check.  The PR changes the check to the explicit option, so we shouldn't need that anymore.